Set/Card/CardViewBuilder.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 import SwiftUI
 
 class CardViewBuilder {
-    func build(for card: CardModel) -> CardView {
-        let viewModel = CardViewModel(card: card)
+    func build(for card: CardModel, _ isColorBlindModeEnabled: Bool) -> CardView {
+        let viewModel = CardViewModel(card: card, isColorBlindModeEnabled)
```

Comment:
Don't use underscore for second attribute

Reply:
fixed